courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -1318,56 +1114,21 @@ Mode `out` Risk for Scalars
 * Any effect upon external objects or external environment
 
    - Typically alteration of non-local variables or states
+   - Can cause hard-to-debug errors
+   - Not legal in SPARK
```

Comment:
This is only true for functions - your example shows a procedure

Reply:
Good catch, it's been in the material for a while.